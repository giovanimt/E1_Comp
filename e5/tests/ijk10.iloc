addI rsp, 4 => rsp
addI rsp, 4 => rsp
addI rsp, 4 => rsp
loadI 400 => r0
addI rfp, 4 => r1
store r0 => r1
loadI 2 => r2
addI rfp, 8 => r3
store r2 => r3
addI rfp, 4 => r4
load r4 => r5
addI rfp, 8 => r6
load r6 => r7
div r5, r7 => r8
addI rfp, 4 => r9
load r9 => r10
addI rfp, 8 => r11
load r11 => r12
div r10, r12 => r13
addI rfp, 8 => r14
load r14 => r15
div r13, r15 => r16
add r8, r16 => r17
addI rfp, 0 => r18
store r17 => r18
