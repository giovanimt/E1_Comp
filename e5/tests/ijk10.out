00015163 300
00015167 400
00015171 2
